rever.xsh

```diff
@@ -0,0 +1,7 @@
+$PROJECT = $GITHUB_REPO = 'cinnabar'
+$GITHUB_ORG = 'OpenFreeEnergy'
+
+$ACTIVITIES = ['changelog']
+
+$CHANGELOG_FILENAME = 'docs/changelog.rst'
```

Comment:
```suggestion
$CHANGELOG_FILENAME = 'docs/CHANGELOG.rst'
```
I renamed the file from `changelog.rst` to `CHANGELOG.rst` just to keep things consistent  between repos.